### PR TITLE
Scoped vars docs

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

File: docs/source/datastore.rst [hunk 1]
```diff
@@ -65,6 +65,50 @@ Delete an existing key value pair.
...
+
+Scoping items stored in datastore
+---------------------------------
+
+By default, all items stored in key value store are stored in ``system`` scope. This
+basically means every user has access to these variables. You'd typically use the
+jinja expression ``{{system.key_name}}`` to refer to these variables in actions or
+workflows. Starting version 1.5 of |st2|, you can now scope variables to a specific
+user. With authentication enabled, you can now control who can read or write into those
+variables. For example, to set variable ``date_cmd`` for the particular user, use
```
Comment (Kami) [inline docs/source/datastore.rst]: particular -> currently authenticated user (or similar)

File: docs/source/datastore.rst [hunk 2]
```diff
@@ -65,6 +65,50 @@ Delete an existing key value pair.
...
+
+::
+
+    st2 key set date_cmd "date +%s"
+
+This variable won't clash with the user variables under same name. Also, you can refer
+to user variables in actions or workflows. The jinja syntax to do so is
+``{{user.date_cmd}}``. Note that the notion of ``user`` is available only when actions
+or workflows are run manually. The notion of ``user`` is non-existent when automations
+are run (actions kicked off via rules). So use of user scoped variables is limited to manual execution of actions or workflows.
```
Comment (Kami) [inline docs/source/datastore.rst]: Maybe good idea to add that this is a current limitation aka limitation as of now, but we plan to address it in the future :P

Comment (lakshmi-kannan) [inline docs/source/datastore.rst]: We don't know if we'd address it.

File: docs/source/datastore.rst (hunk 1 above)
Comment (lakshmi-kannan) [inline docs/source/datastore.rst]: gp. fix incoming.

**Pull request merged**